lib/data_scripts/update_bangladesh_regions_script.rb

```diff
@@ -27,6 +27,7 @@ def initialize(dry_run: true, csv_path: DEFAULT_CSV_PATH)
 
   def call
     destroy_empty_facilities
+    fix_incorrect_regions
```

Comment:
There may be more of these, but wanted to add a little example of how we can fix these up.